src/tts_daemon.py

```
@@ -25,25 +25,150 @@
+        raise ValueError("Reference audio must be a regular file.")
+
+    stats = resolved.stat()
+    if stats.st_size > MAX_REFERENCE_WAV_BYTES:
+        raise ValueError(
+            f"Reference audio exceeds {MAX_REFERENCE_WAV_BYTES // (1024 * 1024)} MB."
+        )
+
+    return resolved
+
+
+def create_app(
```

Ruff (0.15.12)

[warning] Too many statements (86 > 50)

(PLR0915)